django_sharding_library/manager.py

```diff
@@ -50,6 +50,11 @@ def create(self, **kwargs):
 
 class ShardManager(Manager):
 
+    def __init__(self, *args, **kwargs):
+        return_value = super(ShardManager, self).__init__(*args, **kwargs)
+        self.name = 'shard_manager'
```

Comment:
Well, I am using it in 1.10 right now and it hasn't been a problem yet. Definitely set it if you ran into an issue, we are not using all the features that are in there, just the ones we really wanted so we might not have run into that issue even though its a legit one.